framework/wazuh/rbac/default/policies.yaml

```diff
@@ -364,7 +364,7 @@ default_policies:
           - '*:*:*'
         effect: allow
 
-  events_ingest:
+  event_ingest:
```

Comment:
We'd better not change default policies or any other RBAC resource names. Otherwise, fresh deployments post-4.9.0 will have a different policy name for the POST /events endpoint than pre-4.9.0, which could be confusing.